apps/poker-roll/src/main.tsx

```diff
@@ -1,13 +1,41 @@
 import { StrictMode } from 'react';
 import * as ReactDOM from 'react-dom/client';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from "react-router-dom";
+
+// import Cashgame 
+// import Bankroll 
+// import Tournament 
 
 import App from './app/app';
 
-const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
-);
-root.render(
-  <StrictMode>
-    <App />
-  </StrictMode>
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Root />,
+    loader: rootLoader,
+    children: [
+      {
+        path: "/bankroll",
+        element: <Bankroll />,
+        loader: bankrollLoader,
+      },
+      {
+        path: "/cashgame",
+        element: <Cashgame />,
+        loader: cashgameLoader,
+      },
+      {
+        path: "/tournament",
+        element: <Tournament />,
+        loader: tournamentLoader,
+      }
+    ],
+  },
+]);
+
+ReactDOM.createRoot(document.getElementById("root")).render(
```

Comment:
TypeScript's complaining here because it thinks your element might be `null`. Since we know it exists can fix with a "type assertion" to tell TypeScript this will definitely be an html element. There's a built-in type for that called `HTMLElement`:

`document.getElementById("root") as HTMLElement`